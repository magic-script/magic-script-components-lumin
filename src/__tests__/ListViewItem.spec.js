//
import React from "react";
import { create } from "react-test-renderer";
import { ListViewItem } from "magic-script-components";

describe("ListViewItem component", () => {
  test("Matches the snapshot", () => {
    const listviewitem = create(<ListViewItem />);
    expect(listviewitem.toJSON()).toMatchSnapshot();
  });
});
