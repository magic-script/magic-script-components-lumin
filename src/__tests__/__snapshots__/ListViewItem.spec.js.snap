// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListViewItem component Matches the snapshot 1`] = `<listViewItem />`;
